### PR TITLE
Add print_versions

### PR DESCRIPTION
**Pull request by maurelian**
A little QoL improvement for debugging.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -180,7 +180,25 @@ jobs:
...
+    steps:
+      - checkout
+      - run:
+          name: print versions
+          command: |
+            just --version
+            yq --version
+            forge --version
+            rustc --version
+            cargo --version
```
Comment (mds1) [inline .circleci/config.yml:196]: CI is failing with `/bin/bash: line 4: rustc: command not found`. I don't think we require rustc or cargo as deps for this repo

Comment (maurelian) [reply .circleci/config.yml]: 52c3075652c92909091525e7f4b318032ffe342b

Comment (maurelian) [reply .circleci/config.yml]: oh, cargo too. kk

Comment (maurelian) [reply .circleci/config.yml]: OK also: 105e310f9372081fb818daa96df53c24af4cc79c

I made the `just` installation use either `cargo` or `brew`. I think it's fine not to document how to install either of those tools, if a signer has trouble with that step we can just help them directly.

**Review by mds1 — APPROVED** [review]

**Pull request merged**